models/db.py

```diff
@@ -184,7 +184,37 @@ def sanitize_fields(form):
     for site in current.SITES:
         site_handle = site.lower() + "_handle"
         form.vars[site_handle] = form.vars[site_handle].lower()
-
+#-----------------------------------------------------------------------------
+def inside_institute(form):
+        # Send mail to all users form the same college when someone registers
+    to = []
+    for mail in db(utable.email == form.vars.institute).select():
+        to.append(mail)
+    subject = "New user registered from your college"
+    message = """
```

Comment:
Okay sorry here - I didn't mention what should the mail content look like.
I want something like this to be seen in the mails - 
```
Hello <1>,
<2> just joined StopStalk.
Send a friend request now - <3>
To stop receiving mails - <4>
```
where
<1> = row.stopstalk_handle
<2> = form.vars.first_name + " " + form.vars.last_name
<3> = URL('default', 'search')
<4> = URL("default", "unsubscribe", scheme=True, host=True, extension=False) [see here](https://github.com/stopstalk/stopstalk-deployment/blob/master/controllers/default.py#L906)